blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -1,23 +1,81 @@
-Activity=Activity
```

Comment:
IMO it makes sense to have a strict alphabetic order for the keys of our properties. Meaning we should do something like

```
sort -t\  -n Messages.properties > Messages.properties_sort
#review
mv Messages.properties_sort Messages.properties
```

on them before commit.